src/js/iqons.js

```diff
@@ -206,15 +206,15 @@ export default class Iqons {
         // We have four features, but we want only three words to describe them.
         // The order of the three words must be: adjective, verb, noun.
         // That leaves us with
-        const _idx0 =            entropy1       % 4; // feature of adjective
-        const _idx1 = Math.floor(entropy1 / 4)  % 3; // feature of verb
-        const _idx2 = Math.floor(entropy1 / 12) % 2; // feature of noun
+        const _idx0 =            featureSelectors       % 4; // feature of adjective
+        const _idx1 = Math.floor(featureSelectors / 4)  % 3; // feature of verb
+        const _idx2 = Math.floor(featureSelectors / 12) % 2; // feature of noun
```

Comment:
`feature of adjective` or `feature for adjective`?

Reply:
Really not sure. `for` sounds a bit better.